modules/user_manual/pages/files/gallery_app.adoc

```diff
@@ -243,13 +247,20 @@ You can add standard configuration items to the same configuration file:
  native_svg: yes
 ----
 
-[[possible-future-extensions]]
-== Possible Future Extensions
+== Uninstalling Gallery App
```

Comment:
Remove old folder gallery? https://github.com/owncloud/docs/issues/1465#issuecomment-513887603

Reply:
Please see my comment on that comment, @ho4ho.